docs/flow/modin/experimental/xgboost.rst

```diff
@@ -72,27 +72,27 @@ Approximate execution flow of internal implementation is the next:
    * "default_predict" - ``num_actors`` will be computed using condition that 1 actor should use maximum 8 CPUs.
      This condition was chosen to combine  parallelization techniques: parallel actors and parallel threads.
 
-**Note**: ``num_actors`` parameter is advisable to provide to public functions :ref:`train <train>` and ``predict``
-method of :ref:`Booster<Booster>` class to reach best performance for any own case.
+**Note**: ``num_actors`` parameter is made available from public functions :ref:`train <train>` and ``predict``
+method of :ref:`Booster<Booster>` class to allow fine-tuning for best performance in specific use cases.
 
-3. ``ray.util.placement_group`` is created to reserve of all available Ray resources. After that
+3. ``ray.util.placement_group`` is created to reserve all available Ray resources. After that
    :ref:`ModinXGBoostActor <Actor>` objects are created using resources of previously created placement group.
 
-4. Data (`dtrain` for :ref:`_train <internal_train>`, `data` for :ref:`_predict <internal_predict>` is split
+4. Data (`dtrain` for :ref:`_train <internal_train>`, `data` for :ref:`_predict <internal_predict>`) is split
    between actors evenly. Internal function :ref:`_split_data_across_actors <split_data_across_actors>` runs
-   assigning of row partitions to actors using internal function :ref:`_assign_row_partitions_to_actors <assign_parts>`.
-   This function creates dictionary in the form: `{actor_rank: ([part_i0, part_i3, ..], [0, 3, ..]), ..}` - for training,
-   `{actor_rank: [part_i0, part_i1, ..], ..}` - for prediction.
+   assigning row partitions to actors using internal function :ref:`_assign_row_partitions_to_actors <assign_parts>`.
+   This function creates dictionary in the form: `{actor_rank: ([part_i0, part_i3, ..], [0, 3, ..]), ..}` for training,
+   `{actor_rank: [part_i0, part_i1, ..], ..}` for prediction.
 
-**Note**: :ref:`_assign_row_partitions_to_actors <assign_parts>` considers IP addresses of row partitions of `dtrain` data to
-minimize excess data transfer.
+**Note**: :ref:`_assign_row_partitions_to_actors <assign_parts>` takes into account IP addresses of row partitions
+of `dtrain` data to minimize excess data transfer.
 
 5. For each actor object methods ``set_train_data`` or ``set_predict_data`` of :ref:`ModinXGBoostActor <Actor>` class
-   object are called remotely to run loading row partitions belonging to actor according to dictionary
-   from previous step. When data is passed in actor, auto-materialization of row partitions
+   object are called remotely. Those methods run loading row partitions in actor according to dictionary with partitions
```

Comment:
what did you mean by "class object" here?

Reply:
This was duplication of "actor object" from the beginning of sentence:). Sentence was re-phrased.